oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexProviderService.java

```diff
@@ -110,11 +110,11 @@
                 description = "Local file system path where text extraction cache stores/load entries to recover from timed out operation")
         String localTextExtractionDir();
 
-        @AttributeDefinition(name = "Remote index cleanup frequency", description = "Frequency (in seconds) of running remote index deletion scheduled task")
+        @AttributeDefinition(name = "Remote index cleanup frequency", description = "Frequency (in seconds) of running remote index deletion scheduled task. Set this to -1 to disable the task.")
         int remoteIndexCleanupFrequency() default 60;
 
         @AttributeDefinition(name = "Remote index deletion threshold", description = "Time in seconds after which a remote index whose local index is not found gets deleted")
-        int remoteIndexDeletionThreshold() default 300;
+        int remoteIndexDeletionThreshold() default 86400;
```

Comment:
I usually use a formula here, e.g. (5 * 60) instead of (300) or (24 * 60 * 60). Then, I would also document the default value ("Default is 1 day").